parquet-encoding/src/main/java/org/apache/parquet/column/values/bitpacking/BytePacker.java

```diff
@@ -71,6 +73,9 @@ public final int getBitWidth() {
    * @param output the output values
    * @param outPos where to write to in output
    */
+  public abstract void unpack8Values(final ByteBuffer input, final int inPos, final int[] output, final int outPos);
+  
+  //Compatible API
   public abstract void unpack8Values(final byte[] input, final int inPos, final int[] output, final int outPos);
```

Comment:
These implementations are actually currently being code-generated, would you like me to take the results of the generation and try to move them here? Should this abstract class instead use the same code generation?